src/analyses/base.ml

```diff
@@ -443,7 +436,7 @@ struct
       | `Top ->
         let typ = AD.get_type adr in
         let warning = "Unknown value in " ^ AD.short 40 adr ^ " could be an escaped pointer address!" in
-        if is_immediate_type typ then () else M.warn_each warning; empty
+        if ValueDomain.Compound.is_immediate_type typ then () else M.warn_each warning; empty
```

Comment:
Why not VD.is_immediate_type ?

Reply:
No good reason, it was just more readable to me. I'll use VD in the future because that's the style everywhere.